méli-mélo/th-canadaday/style.css

```diff
@@ -0,0 +1,35 @@
+/* Thematic red */
+.bg-canadaday,
+body .bg-canadaday {
+    background-color: #990000;
+}
+
+.panel-canadaday,
+hr.brdr-canadaday {
+    border-color:#990000;
+}
+
+/* Hover and focus state */
+a.bg-canadaday:hover,
+a.bg-canadaday:focus,
+button.bg-canadaday:hover,
+button.bg-canadaday:focus {
+    background-color: #6b0303;
+}
+
+/* Active state */
+a.bg-canadaday:active,
+a.bg-canadaday:active:focus,
+a.bg-canadaday.active:focus,
+a.bg-canadaday.active.focus,
+a.bg-canadaday:active.focus,
+button.bg-canadaday:active {
+    background-color: #000000 !important;
```

Comment:
I don't think you need to use `!important` here. In what situation it is needed?